### PR TITLE
viper.cpp: Game title consistencies, year fixes and note for US Tsurugi title.

### PR DESCRIPTION
**Pull request by mikehyrus**
This a continuation of where I left off in #9184 due to my repo being funky earlier causing a refork.
So yes I’ve seen some videos where the Korean sets used the Japanese Keisatsukan title text in them even though they have Korean and some English for the how to’s and ranks, etc, etc.

### CODE REVIEW

File: src/mame/drivers/viper.cpp [hunk 1]
```diff
@@ -98,7 +98,7 @@
         p911(all)           Goes in-game. Controllers are not emulated. Various graphical glitches.
         tsurugi,j           Goes in-game. Controllers are not emulated. Various graphical glitches.
 
-        gticlub2,ea         Attract mode works. Coins up. Hangs in various places. Will crash with "network error" after stage is selected.
+        gticlub2,drivparty  Attract mode works. Coins up. Hangs in various places. Will crash with "network error" after stage is selected.
```
Comment (cuavas) [inline src/mame/drivers/viper.cpp:101]: The part after the comma in this list is clearly the region that the comment applies to.  Don’t break it.

File: src/mame/drivers/viper.cpp [hunk 2]
```diff
@@ -3429,21 +3429,21 @@ GAME(2001, boxingm,   kviper,    viper,     boxingm,    viper_state, init_viperc
...
+GAME(2000, p911k,     p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "The Keisatsukan: Shinjuku 24-ji (ver KAE)", MACHINE_NOT_WORKING)
+GAME(2000, p911ac,    p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "The Keisatsukan: Shinjuku 24-ji (ver AAC)", MACHINE_NOT_WORKING)
+GAME(2000, p911kc,    p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "The Keisatsukan: Shinjuku 24-ji (ver KAC)", MACHINE_NOT_WORKING)
+GAME(2000, p911ud,    p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "Police 911 (ver UAD)", MACHINE_NOT_WORKING)
+GAME(2000, p247,      p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "Police 24/7 (ver EAD)", MACHINE_NOT_WORKING)
+GAME(2000, p247b,     p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "Police 24/7 (ver EAD, alt)", MACHINE_NOT_WORKING)
+GAME(2000, p911j,     p911,      viper,     p911,       viper_state, init_vipercf,  ROT90, "Konami", "The Keisatsukan: Shinjuku 24-ji (ver JAE)", MACHINE_NOT_WORKING)
```
Comment (cuavas) [inline src/mame/drivers/viper.cpp:3446]: Please don’t change the short names – they’re more manageable when they use suffixes on the parent name.

**Review by cuavas — CHANGES_REQUESTED** [review]

File: src/mame/drivers/viper.cpp (hunk 2 above)
Comment (mikehyrus) [reply src/mame/drivers/viper.cpp:3446]: I'll revert the short names back to the way they were later today.

**Comment by mikehyrus** [comment]
@cuavas
Small names are back to the way the were before I changed them.

File: src/mame/drivers/viper.cpp (hunk 1 above)
Comment (987123879113) [reply src/mame/drivers/viper.cpp]: The comment applies to both `gticlub2` and `gticlub2ea`. Whoever wrote the initial comment just did it in a shorthand lazy way that is meant to be `gticlub2 and gticlub2ea`.

**Comment by mikehyrus** [comment]
Addressed a few days back.

**Comment by mikehyrus** [comment]
@cuavas
Also reverted the one part where it showed the region comment part to the way it was just a couple of hours after the change was requested a few days back.

**Pull request merged**